infrastructure_overview/service_types.md

```diff
@@ -79,7 +79,7 @@ The `extraParameters` provide the addresses of some infrastructure components, f
 
 | Parameter name | Optional | Description |
 --- | --- | :--- |
-| `sessionToken` | yes | Session token to be used throughout the workflow |
+| `sessionToken` | no | Session token to be used throughout the workflow |
```

Comment:
I think we do have synchronous services without a session token in the function signature. Is it really non-optional?

Reply:
Agreed, this is only mandatory for applications, as those need to it to authenticate with the WFM.